widgetry/src/widgets/drag_drop.rs

```diff
@@ -329,6 +337,17 @@ impl<T: 'static + Copy + PartialEq> WidgetImpl for DragDrop<T> {
             self.state = new_state;
             self.recalc_draw(ctx);
         }
+
+        match self.state {
```

Comment:
The 3 minutes I spent doing this was definitely the biggest bang/buck.

Reply:
Nice! I'm noticing some stuttering -- do you see something like this?
![screencast](https://user-images.githubusercontent.com/1664407/132783268-b900e7c1-1100-42f8-a537-d384baf14509.gif)
It's likely due to https://github.com/a-b-street/abstreet/blob/50b8c533bfcb98d79bbb029ec444ce37e871ee17/widgetry/src/runner.rs#L39. IIRC, we reset the cursor every single event, and rely on caller code to keep setting other states. I'm not sure which event is causing this to flicker here; I can take a look tomorrow...